test/system/application-scans/java.spec.ts

```diff
@@ -482,14 +487,19 @@ describe("jar binaries scanning", () => {
         const imageNameAndTag = `docker-archive:${fixturePath}`;
 
         it("should not unpack jars if nested-jars-depth flag is set to 0", async () => {
```

Comment:
I wonder if there's a way with `it.each` that I can run all the tests once with --nested and once with --shaded, would it be easier to maintain, and how hard it would be to break apart once we get rid of --shaded 🤔

Reply:
nope, can't really. you cannot nest `describe` or even `test` inside an it.each, so this structure wont work.

Reply:
https://github.com/snyk/snyk-docker-plugin/pull/392